package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "type": "module",
+  "name": "call_of_duty",
```

Comment:
When you update the repo name, fix it here to `cabab-case`
```suggestion
  "name": "call-of-duty",
```

Reply:
done